contracts/sol6/KyberNetwork.sol

```diff
@@ -119,17 +114,10 @@ contract KyberNetwork is WithdrawableNoModifiers, Utils5, IKyberNetwork, Reentra
     event KyberNetworkSetEnable(bool isEnabled);
     event KyberProxyAdded(address proxy);
     event KyberProxyRemoved(address proxy);
-    event AddReserveToNetwork(
-        address indexed reserve,
-        bytes32 indexed reserveId,
-        IKyberStorage.ReserveType reserveType,
-        address indexed rebateWallet,
-        bool add
-    );
-    event ListReservePairs(
-        address indexed reserve,
-        IERC20 indexed src,
-        IERC20 indexed dest,
+
+    event ListedReservesForToken(
+        IERC20 indexed token,
+        address[] indexed reserves,
```

Comment:
Remove indexed here.... Cos it'll be the hash of the array, which won't be useful for indexing:
https://solidity.readthedocs.io/en/v0.6.6/contracts.html#events

Reply:
oh ok